src/scss/buttons.scss

```diff
@@ -115,3 +115,57 @@
     }
   }
 }
+
+.icon-btn {
```

Comment:
extracted `row-btn` out to a new re-usable style. `row-btn` now extends `icon-btn`.